plugin-api/src/main/java/org/hyperledger/besu/plugin/services/BesuEvents.java

```diff
@@ -29,9 +29,11 @@
 /**
  * This service allows plugins to attach to various events during the normal operation of Besu.
  *
- * <p>Currently supported events
+ * <p>Currently supported events:
  *
  * <ul>
+ *   <li><b>BlockAdded</b> - Fired when a new block has been evaluated and validated.
+ *   <li><b>BlockReorg</b> - Fired when a reorg block has been evaluated and validated.
```

Comment:
reorg blocks are not evaluated and validated.  They are the blocks that are being removed when we change to a different chainhead that is not a direct follow on. perhaps "Fired when a block is removed from the chain in order to change to a new chainhead."?